javascript-custom-methods/calculator.js

```diff
@@ -0,0 +1,32 @@
+/* exported calculator */
+var calculator = {
+  add: function (x, y) {
+    return (x + y);
+  },
+  subtract: function (x, y) {
+    return (x - y);
+  },
+  multiply: function (x, y) {
+    return (x * y);
+  },
+  divide: function (x, y) {
+    return (x / y);
+  },
+  square: function (x) {
+    return (x * x);
+  },
+  sumAll: function (numbers) {
+    var sum = 0;
+    for (var i = 0; i < numbers.length; i++) {
+      sum += numbers[i];
+    } return sum;
+  },
+  getAverage: function (numbers) {
+    var average = [];
```

Comment:
since you are storing a number in this variable, I would give it an initial value of `0`

Reply:
ah makes sense!